### PR TITLE
enh(php) some PHP cleanups

### PR DESCRIPTION
**Pull request by joshgoebel**
- rewrite php keyword lists
- enh(php) improve namespace/use highlighting
- fixup: namespace

### Checklist
- [x] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: src/languages/php.js [hunk 1]
```diff
@@ -71,36 +71,210 @@ export default function(hljs) {
...
+    "SplDoublyLinkedList",
+    "SplFileInfo",
+    "SplFileObject",
+    "SplFixedArray",
+    "SplHeap",
+    "SplMaxHeap",
+    "SplMinHeap",
+    "SplObjectStorage",
+    "SplObserver",
+    "SplObserver",
```
Comment (wkania) [inline src/languages/php.js:233]: Duplicate entry `SplObserver`. Was there before fefactoring.

File: src/languages/php.js [hunk 2]
```diff
@@ -71,36 +71,210 @@ export default function(hljs) {
...
+    "SplPriorityQueue",
+    "SplQueue",
+    "SplStack",
+    "SplSubject",
+    "SplSubject",
```
Comment (wkania) [inline src/languages/php.js:238]: Duplicate entry `SplSubject`. Was there before refactoring.

File: test/markup/php/namespace.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+namespace Location\Web;
+namespace foo;
+use My\Full\Classname as Another;
+use My\Full\NSname;
+use ArrayObject;
+use function My\Full\functionName;
```
Comment (wkania) [inline test/markup/php/namespace.txt:6]: Maybe one more complex example with groupped use declaration?
`use Example\{ClassA, ClassB, ClassC as C};`

Comment (joshgoebel) [reply test/markup/php/namespace.txt:6]: Done.

**Pull request merged**